parcel.go

```diff
@@ -36,25 +63,61 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	query := "SELECT number, client, status, address, created_at FROM parcel WHERE client = :client"
+	rows, err := s.db.Query(
+		query,
+		sql.Named("client", client),
+	)
+	if err != nil {
+		return res, err
```

Comment:
Лучше возвращать `nil`, чтобы не было "полуданных"
```suggestion
		return nil, err
```